RecoPixelVertexing/PixelVertexFinding/src/PixelVertexProducer.cc

```diff
@@ -41,7 +41,7 @@ PixelVertexProducer::PixelVertexProducer(const edm::ParameterSet& conf)
     track_pt_min   = PVcomparerPSet.getParameter<double>("track_pt_min");    
     if (track_pt_min != ptMin_) {
       if (track_pt_min < ptMin_)
-	edm::LogWarning("PixelVertexProducer") << "minimum track pT setting differs between PixelVertexProducer (" << ptMin_ << ") and PVcomparer (" << track_pt_min << ") [PVcomparer considers tracks w/ lower threshold than PixelVertexProducer does] !!!";
+	edm::LogInfo("PixelVertexProducer") << "minimum track pT setting differs between PixelVertexProducer (" << ptMin_ << ") and PVcomparer (" << track_pt_min << ") [PVcomparer considers tracks w/ lower threshold than PixelVertexProducer does] !!!";
```

Comment:
Andrea
 I’m silencing in the message logger using the same-since-ever configuration for production. The issue that no one seems to be aware of this message present during all 2015 production is a sign of how useful it is. Certainly there is plenty of time to understand the consequence now that I’ve raised awareness of it for the 2016 menus. We will continue to have it for 74x and 75x data taking in case further reminders are needed.

> On Oct 21, 2015, at 3:51 PM, Andrea Bocci <notifications@github.com> wrote:
> 
> In RecoPixelVertexing/PixelVertexFinding/src/PixelVertexProducer.cc:
> 
> > @@ -41,7 +41,7 @@ PixelVertexProducer::PixelVertexProducer(const edm::ParameterSet& conf)
> >      track_pt_min   = PVcomparerPSet.getParameter<double>("track_pt_min");    
> >      if (track_pt_min != ptMin_) {
> >        if (track_pt_min < ptMin_)
> > -	edm::LogWarning("PixelVertexProducer") << "minimum track pT setting differs between PixelVertexProducer (" << ptMin_ << ") and PVcomparer (" << track_pt_min << ") [PVcomparer considers tracks w/ lower threshold than PixelVertexProducer does] !!!";
> > +	edm::LogInfo("PixelVertexProducer") << "minimum track pT setting differs between PixelVertexProducer (" << ptMin_ << ") and PVcomparer (" << track_pt_min << ") [PVcomparer considers tracks w/ lower threshold than PixelVertexProducer does] !!!";
> 
> On 21 October 2015 at 15:41, David Lange <notifications@github.com> wrote: we really need more than 5 billion MC events worth of the printout?
> As I did write: no. You can silence it in the MessageLogger configuration. I don't think it makes sense to make the change in the HLT menu now - the next occasion to review the impact of the change (though none is expected) would be next Thursday, with only few days of data taking left. In any case, do not downgrade the LogWarning to a LogInfo, just because it is doing its job of catching the problem it was intended to catch. .Andrea
> —
> Reply to this email directly or view it on GitHub.
>